kratos/processes/replace_elements_and_condition_process.h

```diff
@@ -53,33 +52,34 @@ class ReplaceElementsAndConditionsProcess : public Process
     ///@}
     ///@name Life Cycle
     ///@{
-    ReplaceElementsAndConditionsProcess(ModelPart& model_part, 
-                              Parameters Settings
-                                   ) : Process(Flags()) , mr_model_part(model_part), mSettings( Settings)
+    
+    /**
+     * @brief Default constructor
+     * @param rModelPart The model part where to assign the conditions and elements
+     * @param Settings The parameters containing the names of the conditions and elements
+     */
+    ReplaceElementsAndConditionsProcess(
+        ModelPart& rModelPart, 
+        Parameters Settings
+        ) : Process(Flags()) , 
+            mrModelPart(rModelPart), 
+            mSettings( Settings)
     {
         KRATOS_TRY
 
-//only include validation with c++11 since raw_literals do not exist in c++03
-#if __cplusplus >= 201103L
```

Comment:
Just to state. This is no longer needed because we use C++11 as default ( in case someone searches why we deleted this )

Reply:
yep, I know, that why I remove it